ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/DefaultSynchronizer.java

```diff
@@ -205,6 +213,9 @@ public TrailingPeerRequirements calculateTrailingPeerRequirements() {
   public CompletableFuture<Void> start() {
     if (running.compareAndSet(false, true)) {
       LOG.info("Starting synchronizer.");
+
+      syncDurationMetrics.startTimer(SyncDurationMetrics.Labels.TOTAL_SYNC_DURATION);
```

Comment:
does startTimer work as intended if it has been stopped?  Is it additive, or does it replace the prior metric values?

Thinking about the worldstate heal re-use of the synchronizer

Reply:
startTimer() only works the first time it is called. After the first start the call has no effect.